deployer/README.md

```diff
@@ -1175,3 +1175,260 @@ the first created. You can check if the route in question is defined in multiple
     logging     kibana-ops   kibana-ops.example.com              logging-kibana-ops
 
 (In this example there are no overlapping routes.)
+
+## Splitting your Fluentd for high scale capacity (Advanced)
```

Comment:
Agreed, that would be a nice thing to be able to configure with our logging role, if this is an approach we'll keep around